src/timeseriesflattener/flattener.py

```diff
@@ -97,7 +98,9 @@ class Flattener:
     compute_lazily: bool = False
     n_workers: int | None = None
 
-    def aggregate_timeseries(self, specs: Sequence[ValueSpecification]) -> AggregatedFrame:
+    def aggregate_timeseries(
+        self, specs: Sequence[ValueSpecification], timedelta_days: Union[dt.timedelta, None] = None
```

Comment:
```suggestion
        self, specs: Sequence[ValueSpecification], timedelta_days: dt.timedelta | None = None
```

Reply:
Also, rename `timedelta_days` in all locations to something like "step width"?

Reply:
hmm, the resolution is still days though, don't we want to include that?